client/src/control4/stages/copyToOutputStage.ts

```diff
@@ -17,9 +17,7 @@ export default class CopyToOutputStage implements BuildStage {
         return new Promise(async (resolve, reject) => {
             try {
                 if (vscode.workspace.getConfiguration('control4.build').get<boolean>('exportToDriverLocation')) {
-                    let root = (this.pkg.control4 && this.pkg.control4.agent) ? 

```

Comment:
This appears to be the only place `pkg.control4.agent` is used.

I've left it defined in the `Control4Package` class, not sure if you have other uses intended for it.

Reply:
pkg.control4.agent was just meant to be a boolean to determine if the driver should be configured as an agent (and where to put it). I believe the directory no longer matters and Control4 resorted to just having it set in the XML.

```xml
<agent>true</agent>
```